utils/tokenutils.go

```diff
@@ -0,0 +1,96 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+type Claims struct {
+	NotBefore int64 `json:"nbf"`
```

Comment:
`jwt.StandardClaims` already has a `NotBefore` field

Reply:
Missed that somehow.  Good catch. Changed the structure to use the field from `jwt.StandardClaims`